harper-core/src/linting/plural_conjugate.rs

```diff
@@ -23,7 +23,7 @@ impl Default for PluralConjugate {
             .then_whitespace()
             .then_exact_word("are");
 
-        let pat = EitherPattern::new(vec![Box::new(plural_case), Box::new(non_plural_case)]);
+        let pat = plural_case.or(Box::new(non_plural_case));
```

Comment:
I wouldn't advise using `case` to refer to `singular` vs `plural` as `case` has a specific meaning in grammar and linguistics: I, we he, she, and they are `subject case`; me, us him, her, and them are `object case`; and you and it are both subject and object case. The correct technical term is `grammatical number` so `plural_number` and `singular_number`.